exercises/game-of-life/canonical-data.json

```diff
@@ -0,0 +1,75 @@
+{
+  "exercise": "game-of-life",
+  "cases": [
+    {
+      "uuid": "ae86ea7d-bd07-4357-90b3-ac7d256bd5c5",
+      "description": "empty matrix",
+      "property": "gameOfLife",
```

Comment:
In terms of exercise design, I'd like to have a different name for the property.

If the implementation of the exercise gets created via a generator we'd end up with both the top level namespace and the property under test being _game of life_. I'd like to start with names that are a reasonable default. People can always choose to override this in an individual track, but starting with a good default helps tremendously.

I'm trying to think of names that have to do with the flow of generations, or population... maybe `tick` (the generations tick by)?

Reply:
tick, round, simulate, advance, next

Any or all of those sounds good to me

Reply:
I like all of those too